python/README.md

```diff
@@ -0,0 +1,32 @@
+# Apache Spark
+
+Spark is a fast and general cluster computing system for Big Data. It provides
```

Comment:
well we don't necessarily want to full Spark README for PySpark so idk. But if we would be ok with the full README we could symlink it as normal.